docs/integrations/source_version_control.md

```diff
@@ -29,3 +29,15 @@ onwards).
 [black-tags]: https://github.com/psf/black/tags
 [pre-commit-mutable-rev]:
   https://pre-commit.com/#using-the-latest-version-for-a-repository
+
+Pre-commit collects all filepaths recursively throughout the repo to pass to Black. You
+can use the `--force-exclude` flag to ignore specific files or directories, which
+pre-commit takes as an argument. See the partial example below ignoring a `tests`
+directory:
+
+```yaml
+hooks:
+  - id: black
+    args:
+      - --force-exclude=tests/
```

Comment:
is this the best place to put this? if you're using pre-commit, you might as well use pre-commit's own `exclude`

I think the more common use-case for `force-exclude` is for putting it in `pyproject.toml`, so that this config stay synced between when running `black` via `pre-commit` or via the command-line

Reply:
I agree with @MarcoGorelli, using `--force-exclude` in pre-commit-config.yaml is a bit idiosyncratic. Either you use it in `pyproject.toml` or you use pre-commit's own exclude configuration. I'd personally recommend pre-commit's exclude first, and then --force-exclude, as the latter can be overkill.